pkg/operator/controller/status/controller.go

```diff
@@ -268,6 +279,18 @@ func (r *reconciler) getOperatorState(ingressNamespace, canaryNamespace string)
 		state.IngressControllers = ingressList.Items
 	}
 
+	// Get all CRDs from the gateway.networking.k8s.io group.
+	clusterCRDs := &apiextensionsv1.CustomResourceDefinitionList{}
+	if err := r.client.List(context.TODO(), clusterCRDs); err != nil {
```

Comment:
If you can use the cache, that reduces load on the API:
```suggestion
	if err := r.cache.List(context.TODO(), clusterCRDs); err != nil {
```

Reply:
Fixed